contracts/token/ERC20_base.cairo

```diff
@@ -130,8 +130,8 @@ func ERC20_transferFrom{
         amount: Uint256
     ) -> ():
     alloc_locals
```

Comment:
is this needed?

Reply:
It's needed here as well. The `syscall_ptr` is defined in the allowances read but is revoked on the `_transfer` invocation otherwise.